python/ray/services.py

```diff
@@ -1253,48 +1276,6 @@ def start_plasma_store(node_ip_address,
     return process_info
 
 
-def start_worker(node_ip_address,
```

Comment:
This currently isn't used though I could see us wanting to use it in the future. Might be worth leaving. Note that there is a corresponding `start_worker` in `node.py`.